src/data_egress/sqs_listener.py

```diff
@@ -189,15 +189,37 @@ def get_to_be_processed_s3_prefixes(message):
 
 
 def query_dynamodb(s3_prefixes, dynamodb):
-    """Query  DynamoDb status table for a given correlation id.
+    """Scan DynamoDB table for generic source prefixes.
+
+    Query  DynamoDb status table for a given correlation id.
 
     Arguments:
         s3_prefixes (string): source bucket prefixes to query dynamo db table
     """
+    generic_source_prefixes = []
     table = dynamodb.Table(DATA_EGRESS_DYNAMO_DB_TABLE)
+    scan_response = table.scan(ProjectionExpression=HASH_KEY)
```

Comment:
Not sure if it would ever affect this table due to its size but worth noting that scan results are paginated, so you may want to check the LastEvaluatedKey in the result and go again if present.